src/WebBrowser-Core.package/WebBrowser.class/class/webbrowserClassForPlatform.st

```diff
@@ -0,0 +1,8 @@
+private - accessing
+webbrowserClassForPlatform
+
+	Smalltalk os isUnix ifTrue: [ ^WBUnixWebBrowser ].
+	Smalltalk os isWindows ifTrue: [ ^WBWindowsWebBrowser ].
+	(Smalltalk os isMacOS or: [ Smalltalk os isMacOSX ])  
+		ifTrue: [ ^WBMacWebBrowser ].
```

Comment:
Wouldn't it be better to make all those classes inherit from the same abstract class and then have something like this?
```smalltalk
WBAstractWebBrowser>>webBrowserClassForPlatform
  ^ self allSubclasses detect: #isForCurrentSystem ifNone: [ self error: 'No web browser class for platform provided' ]
```

```smalltalk
WBUnixWebBrowser>>isForCurrentSystem
  ^ Smalltalk os isUnix
```

Like that it would be easier to extend the system to add other platforms if it is needed in the future.